src/main/java/uk/gov/hmcts/reform/orgrolemapping/domain/service/ParseRequestService.java

```diff
@@ -23,7 +23,7 @@
     public void validateUserRequest(UserRequest userRequest) {
         //parse the user List and validate each user Id to be valid string
         userRequest.getUsers().forEach(user ->
-                ValidationUtil.validateId(NUMBER_TEXT_HYPHEN_PATTERN, user));
+                ValidationUtil.validateId(UUID_PATTERN, user));
```

Comment:
The user can be any number, string and uuid, thats why we added number_text_hyphan_pattern.

Reply:
ok i will reintroduce it